packages/reports/addon/components/report-actions/export.ts

```diff
@@ -59,9 +61,10 @@ export default class ReportActionExport extends Component<Args> {
     const serializedRequest = this.args.model.request.serialize() as RequestV2;
 
     try {
-      const url: string = yield taskFor(this.facts.getDownloadURL).perform(serializedRequest, {
+      let url: string = yield taskFor(this.facts.getDownloadURL).perform(serializedRequest, {
```

Comment:
not sure if we want to pass it to `this.facts.getDownloadURL` or just do
```this.downloadURLLink(`${url}&filename=${this.filename}`)```
I'm fine with either

Reply:
I kept in filename becoz i thought that we can have common across everywhere we can pass it as attribute in stead of appending in the url

Reply:
```suggestion
      const url: string = yield taskFor(this.facts.getDownloadURL).perform(serializedRequest, {
```